packages/contractkit/src/base.ts

```diff
@@ -15,10 +15,12 @@ export enum CeloContract {
   GasPriceMinimum = 'GasPriceMinimum',
   GoldToken = 'GoldToken',
   Governance = 'Governance',
+  GovernanceApproverMultiSig = 'GovernanceApproverMultiSig',
```

Comment:
Probably fine to just have a single `MultiSig` here, given the contract code is the same for all of them. Ditto everywhere else

Reply:
The actual contract is different though... there is a GovernanceApproverMultiSig.sol and ReserveSpenderMultiSig.sol.  they both extend multisig, so it seems it would work, just feels a bit inconsistent

Reply:
i've updated to change to just generic multisig